aws/scenarios/vm/os/os.go

```diff
@@ -2,42 +2,24 @@ package os
 
 import (
 	"github.com/DataDog/test-infra-definitions/aws"
-)
-
-type Architecture string
-
-const (
-	AMD64Arch = Architecture("x86_64")
-	ARM64Arch = Architecture("arm64")
-)
-
-type OSType int
-
-const (
-	WindowsOS OSType = iota
-	UbuntuOS         = iota
-	MacOS            = iota
+	"github.com/DataDog/test-infra-definitions/common/os"
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Could help renaming this to `commonos` to avoid confusion with the current package name

Reply:
Code updated.